<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPassword</class>
 <widget class="QDialog" name="DlgPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="laInfo">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please set a password for this tournament.&lt;/p&gt;&lt;p&gt;The password will be used for authentication against the server that provides read-access to your tournament data through the internet.&lt;/p&gt;&lt;p&gt;Despite the password, data will be transmitted and stored &lt;span style=&quot; font-style:italic;&quot;&gt;unencrypted&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="laRules">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Use at least six characters with at least one number and one letter.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="laCurPw">
         <property name="text">
          <string>Current password: </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="laReapeatPw">
         <property name="text">
          <string>Repeat:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leCurPw">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leRepeatPw">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="laNewPw">
         <property name="text">
          <string>New password: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leNewPw">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="laHint">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>41</green>
             <blue>41</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>41</green>
             <blue>41</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOkay">
         <property name="text">
          <string>Okay</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leCurPw</tabstop>
  <tabstop>leNewPw</tabstop>
  <tabstop>leRepeatPw</tabstop>
  <tabstop>btnOkay</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgPassword</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOkay</sender>
   <signal>clicked()</signal>
   <receiver>DlgPassword</receiver>
   <slot>onBtnOkayClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCurPw</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgPassword</receiver>
   <slot>onLineEditContentChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leNewPw</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgPassword</receiver>
   <slot>onLineEditContentChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leRepeatPw</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgPassword</receiver>
   <slot>onLineEditContentChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLineEditContentChange()</slot>
  <slot>onBtnOkayClicked()</slot>
 </slots>
</ui>
